src/containers/guest-home-page/registration-dialog/RegistrationDialog.tsx

```
@@ -87,6 +88,12 @@
       console.log('User registered with role:', data.role)
       try {
         closeModal()
+        
```

ESLint

[error] Delete `········`

(prettier/prettier)